packages/actionmenu/src/react/item.js

```diff
@@ -134,8 +136,7 @@ export const Item = forwardRef(
 
 Item.displayName = 'ActionMenu.Item'
 Item.defaultProps = {
-  tagName: tagName.a,
-  origin: origins.topRight
```

Comment:
I believe there should be a default origin and it should be topLeft.  Does this change enable that? 

I'm having a hard time telling.  Does origin come from anywhere besides a prop? Doesn't seem like it.

Reply:
Fixed in a different PR